src/test/java/duke/DeadlineTaskTest.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Shouldn't the static import come before the regular import? 
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;
```